test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -1115,7 +1135,7 @@ func (p *namespacePreparer) prepare() error {
 		n := base.DeepCopy()
 		n.Name = fmt.Sprintf("%s-%d", p.prefix, i)
 		if err := testutils.RetryWithExponentialBackOff(func() (bool, error) {
-			_, err := p.client.CoreV1().Namespaces().Create(context.TODO(), n, metav1.CreateOptions{})
+			_, err := p.client.CoreV1().Namespaces().Create(context.Background(), n, metav1.CreateOptions{})
```

Comment:
Why change `context.TODO()` to `context.Background()`?

Reply:
Because of this https://github.com/kubernetes/kubernetes/pull/107771#discussion_r793492551